asciidoc/modules/con_mg_migrating-clusters.adoc

```diff
@@ -0,0 +1,40 @@
+[id="migrating_clusters_{context}"]
+
+= Migrating clusters
+
+It is important to decide the migration strategy for your codebase in a cluster. This is because you cannot add {VertX} {v3x} nodes and {VertX} {v4} nodes together in a single cluster for the following reasons: 
```

Comment:
```suggestion
It is important to decide the migration strategy for your codebase. This is because you cannot add {VertX} {v3x} nodes and {VertX} {v4} nodes together in a single cluster for the following reasons:
```